dev/links.md

```diff
@@ -0,0 +1,4 @@
+| Link                                                | Description                                                  | Tags         |
+| --------------------------------------------------- | ------------------------------------------------------------ | ------------ |
+| [Learn X in Y Minutes](https://learnxinyminutes.com)| Learn things language and cs topics quickly                  | Learning     |
+| [Go By Example](https://gobyexample.com/            | Hands-on introduction to Go using annotated example programs | Learning     |
```

Comment:
```suggestion
| [Go By Example](https://gobyexample.com/)            | Hands-on introduction to Go using annotated example programs | Learning     |
```

Reply:
Ope -- good catch.

Reply:
Edited my PR to include it -- thanks @3ventic, I am trying out Jetbrain's Fleet and it looks like it doesn't autoclose  parens 😭